src/Expensify.js

```diff
@@ -48,17 +38,20 @@ Onyx.registerLogger(({level, message}) => {
 });
 
 const propTypes = {
-    /* Onyx Props */
-
```

Comment:
session is still an Onyx Prop, I think this was moved accidentally?

Reply:
🤦